packages/engine/Specs/Scene/Model/ModelSpec.js

```diff
@@ -715,6 +719,71 @@ describe(
       });
     });
 
+    it("transforms property textures with KHR_texture_transform", async function () {
+      const resource = Resource.createIfNeeded(
+        propertyTextureWithTextureTransformUrl
+      );
+      // The texture in the example model contains contains 8x8 pixels
+      // with increasing 'red' component values [0 to 64)*3, interpreted
+      // as a normalized `UINT8` property.
+      // It has a transform with an offset of [0.25, 0.25], and a scale
+      // of [0.5, 0.5].
+      // Create a custom shader that will render any value that is smaller
+      // than 16*3 or larger than 48*3 (i.e. the first two rows of pixels
+      // or the last two rows of pixels) as completely red.
+      // These pixels should NOT be visible when the transform is applied.
+      const customShader = new CustomShader({
+        fragmentShaderText: `
+          void fragmentMain(FragmentInput fsInput, inout czm_modelMaterial material)
+          {
+            float value = float(fsInput.metadata.exampleProperty);
+            float i = value * 255.0;
+            if (i < 16.0 * 3.0) {
+              material.diffuse = vec3(1.0, 0.0, 0.0);
+            } else if (i >= 48.0 * 3.0) {
+              material.diffuse = vec3(1.0, 0.0, 0.0);
+            } else {
+              material.diffuse = vec3(0.0, 0.0, 0.0);
+            }
+          }
+          `,
+      });
+
+      const gltf = await resource.fetchJson();
+      await loadAndZoomToModelAsync(
+        {
```

Comment:
In light of our slack discussion and your comment on the PR, I think you should add `incrementallyLoadTextures: false` in the options here. I would also add a short comment explaining it's because the test relies on the texture values so we need to ensure the texture is loaded.

Reply:
Done via https://github.com/CesiumGS/cesium/pull/11709/commits/e6026af4ee023bf46db38b824b7af24bea80bed3